### ISSUE TITLE
Cannot continue dino training without destroying performance. 

### ISSUE THREAD

**Issue by mhamilton723** [facebookresearch/dino#207]
Hey, thanks for your great work. I'm looking to fine-tune the dino backbone on a new dataset but cannot get it to train properly, even on imagenet. I've added the following lines to main_dino around line 200:

```
  state_dict = torch.load("checkpoints/dino_deitsmall16_pretrain_full_checkpoint.pth")
  student_dict = {k.replace("module.", ""): v for k, v in state_dict["student"].items()}
  teacher_dict = {k.replace("module.", ""): v for k, v in state_dict["teacher"].items()}
  student.load_state_dict(student_dict)
  teacher.load_state_dict(teacher_dict)
  print("Loaded pretrained model")
```

I've also ensured all the parameters are set to what that checkpoint has. The loss begins at 5 or so, and works it's way lower over time but this always destroys the linear probe performance of the pretrained checkpoint. Am i missing something here in order to re-hydrate training?

**Comment by jafarinia** [comment]
maybe your dataset is small?
